### PR TITLE
Star Schema Benchmark

### PR DESCRIPTION
**Pull request by sdressler**

### CODE REVIEW

**Comment by jukoson** [comment]
Documentation/ Wikipage need updating with the added benchmark

**Comment by sdressler** [comment]
If there are no objections, I am going to merge tomorrow.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,8 @@ This toolset runs TPC-H and TPC-DS benchmarks.
```
Comment (jukoson) [inline README.md:8]: First line of the readme should include a reference to SSB as well, or no listing of the benchmarks provided at all

Comment (sdressler) [reply README.md:8]: Done. Maybe at some point we can rename the whole thing to "s64-benchmark-toolkit" or similar.

**Pull request merged**